package.json

```diff
@@ -132,6 +132,7 @@
 		"rn-root-view": "1.0.3",
 		"semver": "7.3.5",
 		"ua-parser-js": "^1.0.2",
+		"uri-js": "^4.4.1",
```

Comment:
We should consider replacing this lib with the other ones, if this one is better.